README.md

```diff
@@ -66,6 +66,10 @@ Here are some example models that can be downloaded:
 
 > Note: You should have at least 8 GB of RAM available to run the 7B models, 16 GB to run the 13B models, and 32 GB to run the 33B models.
 
+## Suggest new models
+
+To suggest a new model, please open an issue on github and us as label "model request".
```

Comment:
Possible to add this to the `Note:` above? "Trying to run an unsupported model? [Create an issue](https://github.com/ollama/ollama/issues/new?labels=model%20request)"